src/cli/telemetry/telemetry.ts

```diff
@@ -124,10 +124,12 @@ export const prepareData = async (
     component_count,
     targets,
     packages,
+    packages_no_versions,
     arguments: config.flags.args,
     task: config.flags.task,
     stencil,
-    system,
+    system: system,
```

Comment:
Non-blocking, any reason this can't stay as it is today?
```suggestion
    system,
```